ras_party/views/account_view.py

```diff
@@ -113,6 +113,21 @@ def change_respondent_enrolment_status():
     return make_response(jsonify('OK'), 200)
 
 
+@account_view.route('/respondents/disable-user-enrolments', methods=['PUT'])
+def disable_user_enrolments():
+    """Disable all enrolments for a specific respondent email address"""
+    try:
+        email = request.get_json()['email']
+    except TypeError:
+        raise BadRequest('JSON payload not provided')
+    except KeyError:
+        raise BadRequest("Email key must be provided in the JSON payload")
+
+    removed_enrolment_count = account_controller.disable_all_respondent_enrolments(email)
+
+    return make_response(jsonify(f'{removed_enrolment_count} enrolments removed'), 200)
```

Comment:
```suggestion
    return make_response(jsonify({"message": f'{removed_enrolment_count} enrolments removed'}), 200)
```

Responses should be in json format